drupal8.conf

```diff
@@ -66,6 +66,10 @@ server {
             try_files $uri @drupal;
         }
 
+        location ~ "^/sitemaps/[\d]+/sitemap.xml$" {
```

Comment:
I think better to explore more common way https://github.com/wodby/drupal-nginx/blob/master/templates/drupal-8.conf.tpl#L80